scripts/src/report/report_info.py

```diff
@@ -0,0 +1,86 @@
+
+import os
+import sys
+import docker
+import logging
+import json
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+REPORT_ANNOTATIONS = "annotations"
+REPORT_RESULTS = "results"
+REPORT_DIGESTS = "digests"
+REPORT_METADATA= "metadata"
+
+def getReportInfo(report_path,info_type,profile_type,profile_version):
+
+    docker_command = "report " + info_type + " charts/"+os.path.basename(report_path)
+
+    set_values = ""
+    if profile_type:
+        set_values = "profile.vendortype="+profile_type
+    if profile_version:
+        if set_values:
+            set_values = set_values + ","
+        set_values = set_values + "profile.version="+profile_version
+
+    if set_values:
+        docker_command += " --set " + set_values
```

Comment:
be consistent above you do x = x + b and here you use x += b. But if you switch to string formatting it would be something like this:
docker_command = "%s --set %s" % (docker_command, set_values)

Reply:
done.